aws/cleanup.py

```diff
@@ -54,12 +55,21 @@ def default_ctor(_dummy, tag_suffix, node):
     if account_id != config['lambda_account_id']:
         exit(f'The terminator must be run from the lambda account: {config["lambda_account_id"]}')
 
-    cleanup(args.stage, check=args.check, force=args.force, api_name=api_name, test_account_id=test_account_id)
+    if args.region in ('all', 'a'):
+        aws_regions = get_regions()
+    else:
+        aws_regions = [args.region]
+    for region in aws_regions:
+        cleanup(args.stage, check=args.check, force=args.force, api_name=api_name, test_account_id=test_account_id, region=region)
 
 
 def parse_args():
     parser = argparse.ArgumentParser(description='Terminate or destroy stale resources in the AWS test account.')
 
+    parser.add_argument('-r', '--region',
+                        default='us-east-1',
```

Comment:
Since action='append' always results in us-east-1 (or whatever the environment var) being the first item in the list, I've used +nargs instead. But I'm not super familiar with argparse, so there may be a nicer way to do it.